README.md

```diff
@@ -31,6 +31,17 @@ If you are the current maintainer of the package then create a new request using
 * The issue will get labelled as assigned, awaiting approval
 * When the package gets approved by the moderation process, close the issue (or ask for it to be closed).
 
+When you pick up a request for either a new package or a new maintainer, remember to keep the maintainers of this repository updated on the current status of the work being done.
+A comment from the one picking up a request is expected to be done at the following stages of a request:
+
+1. When you start working on the package, or get added as a new maintainer (issue will then be labeled as `Status: In Progress` and you will be assigned to the issue).
+2. When the new package or updated version is pushed to chocolatey (issue will then be moved to `Status: Review`).
+3. When the package is approved by a chocolatey moderator (issue will then be closed and labeled with `Status: Published`).
+
+A maintainer of the repository may at times add a comment on the issue asking for the status of the request when the above have not been followed.
+When this happens, please respond on the issue with the current status of the request as soon as possible.
+Failure to respond within 7-8 days after the comment will be cause the maintainer that picked up the request to be considered unresponsive, and may result in the request being moved back to `Status: Available for Maintainer(s)`.
+
 **Note**: If you are taking over maintenance for a package, you also need to be added ot the list of maintainers. Use the contact admins link from the package page, point to the particular issue in this repository, and ask to be added.
```

Comment:
Typo 'ot' should be to.

And also "Use the 'Contact Site Admins' link on the package page, and point to ..."

Reply:
Although not part of the initial changes, I fixed this as well :smile: